packages/arb-token-bridge-ui/src/components/TransferPanel/Routes/OftV2Route.tsx

```diff
@@ -51,8 +77,9 @@ export function OftV2Route() {
       bridgeIconURI={'/icons/layerzero.svg'}
       durationMs={5 * 60 * 1_000} // 5 minutes in miliseconds
       amountReceived={amount.toString()}
-      isLoadingGasEstimate={isLoading}
+      isLoadingGasEstimate={status !== 'success'}
```

Comment:
We should distinguish between `loading` vs `failure` states. This will show a loader even if the gas estimation has failed.

Reply:
Good catch. I changed the logic to only be true when status is equal to `loading`. Then if it's successful and we have gas cost, it will display it, otherwise, it will show N/A (because gasCost is undefined)

Reply:
![image](https://github.com/user-attachments/assets/880fc63f-c739-4849-a11e-00f30a014d00)